virt/virtual_machines/virt-create-vms.adoc

```diff
@@ -33,8 +33,7 @@ Refer to the virtual machine wizard fields section when running the web console
 include::modules/virt-vm-wizard-fields-web.adoc[leveloffset=+2]
 
 Enable the xref:../../scalability_and_performance/using-cpu-manager.adoc#using-cpu-manager[CPU Manager] to use the high-performance workload profile.
-
-include::modules/virt-networking-wizard-fields-web.adoc[leveloffset=+2]
+include::modules/virt-networking-wizard-fields-web.adoc[leveloffset=+3]
```

Comment:
This include isn't working. 

![image](https://user-images.githubusercontent.com/25209155/228269833-20f91d92-edbc-4e32-8b35-70779a72f962.png)